tests/end2end-tests/data_process/data_gen.py

```diff
@@ -33,8 +33,13 @@
 # without labels, 4) features of different types, 5) node features
 # with different dimensions.
 # We create multiple edges in a similar way.
+def gen_rand_nid(max_nid, num_nodes):
+    node_ids = np.unique(np.random.randint(0, max_nid, num_nodes))
+    if len(node_ids != num_nodes)
+        return gen_rand_nid(max_nid, num_nodes)
```

Comment:
can we print a message to inform people?

Reply:
added